samples/README.md

```diff
@@ -0,0 +1,125 @@
+# Samples
```

Comment:
I'd more specific about what the samples are about in the heading.

Reply:
Kind of long, I'm open to other suggestions:
```suggestion
# Instrumenting Sample Applications with the OpenTelemetry .NET Auto-Instrumentation
```